### PR TITLE
Add badges to README.md

### PR DESCRIPTION
**Pull request by chrismedrela**
Add code coverage and code quality badges to `README.md`.

Enabling code quality badge requires setting up landscape.io for swcarpentry. I've already set it up for my fork and it works: https://github.com/chrismedrela/amy/tree/badges-chrismedrela. I can set it up for swcarpentry, just let me know if it's ok to give landscape.io access to swcarpentry repos.

As far as code coverage is concerned, I can see everything (like travis configuration) is set up, so I guess we just need to change something in coveralls.io service settings.

### CODE REVIEW

File: .landscape.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+strictness: medium
+uses:
+  - django
+python-targets:
+  - 3
```
Comment (gvwilson) [inline .landscape.yaml]: Add end-of-line character.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 ![](workshops/static/amy-logo.png)
 
-![](https://travis-ci.org/swcarpentry/amy.svg?branch=master)
+[![Build Status](https://travis-ci.org/swcarpentry/amy.svg?branch=master)](https://travis-ci.org/swcarpentry/amy)
+[![Code Quality](https://landscape.io/github/swcarpentry/amy/master/landscape.png)](https://landscape.io/github/swcarpentry/amy)
+[![Coverage Status](https://coveralls.io/repos/swcarpentry/amy/badge.png)](https://coveralls.io/github/swcarpentry/amy)
```
Comment (gvwilson) [inline README.md]: Looks good to me - @pbanaszkiewicz ?

**Comment by pbanaszkiewicz** [comment]
@gvwilson I requested access to `swcarpentry` from both landscape.io and coveralls.io - please let me know if / when you accept

**Comment by gvwilson** [comment]
What are these both, and who will manage/sustain/document them?

Thanks,

Greg

**Pull request merged**